qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -219,14 +223,14 @@
           {
             "title": "Demographics",
             "fields": [
+              "age",
               "gender",
-              "country",
+              "nationality",
               "province",
               "city",
-              "age",
+              "country",
```

Comment:
this order doesn't make sense to me (province, city, country)

and there is a lot more data for country than for nationality, so i would also place nationality after country/province/city

Reply:
But nationality is more person centric which I grouped it with age and gender. Then I did location centric after.

Reply:
your call 🙂 but can we do `country, province, city` or `city, province, country` instead of `province, city, country`